### PR TITLE
OCPBUGS-6268 updated the hardware enablement guide

### PR DESCRIPTION
**Pull request by sheriff-rh**
Version(s):
4.12+

Issue:
https://issues.redhat.com/browse/OCPBUGS-6268

Link to docs preview (VPN required):
[About specialized hardware and driver enablement](http://file.rdu.redhat.com/antaylor/OCPBUGS-6268/hardware_enablement/about-hardware-enablement.html)
[Driver Toolkit](http://file.rdu.redhat.com/antaylor/OCPBUGS-6268/hardware_enablement/psap-driver-toolkit.html)

QE review:
- [x] QE has approved this change.

Additional information:
Updated the DTK docs to reflect [upstream](https://github.com/openshift/driver-toolkit/blob/master/README.md).

### CODE REVIEW

**Comment by sheriff-rh** [comment]
@dagrayvid hi! I have updated the DTK docs to be more in-line with [upstream](https://github.com/openshift/driver-toolkit/blob/master/README.md). I noticed there are [some steps (2 and 3)](https://docs.openshift.com/container-platform/4.12/hardware_enablement/psap-driver-toolkit.html#create-simple-kmod-image_driver-toolkit) in the OCP docs that aren't upstream. Are they still needed?

**Comment by dagrayvid** [comment]
Thanks @sheriff-rh, I agree this fix is needed! Since I'm no longer maintaining / owning DTK, we should probably let the current owners review this.

cc @ybettan

File: hardware_enablement/psap-driver-toolkit.adoc [hunk 1]
```diff
@@ -6,7 +6,7 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-Learn about the Driver Toolkit and how you can use it as a base image for driver containers for enabling special software and hardware devices on Kubernetes.
+Learn about the Driver Toolkit and how you can use it as a base image for driver containers for enabling special software and hardware devices on {product-title} deployments.
```
Comment (ybettan) [inline hardware_enablement/psap-driver-toolkit.adoc:9]: We should maybe remove the `psap` prefix to the file name.

Comment (sheriff-rh) [reply hardware_enablement/psap-driver-toolkit.adoc:9]: I renamed the the files `he-driver-toolkit.adoc`. Thanks!

**Comment by sheriff-rh** [comment]
@lalon4 PTAL, thanks!

**Comment by sheriff-rh** [comment]
@cdvultur please review when you have a chance. Thanks!

File: modules/psap-driver-toolkit-pulling.adoc [hunk 2]
```diff
@@ -28,7 +28,11 @@ The driver-toolkit image for the latest minor release will be tagged with the mi
 +
 [source,terminal]
 ----
-$ oc adm release info 4.12.0 --image-for=driver-toolkit
+# For x86 image:
```
Comment (cdvultur) [inline modules/psap-driver-toolkit-pulling.adoc:31]: Wondering if we should add a not about specifying the release, and expecialy about the `.0` , which will evolve with the lifetime of 4.12 . 
Something like: _Update the `.0` with the current version of your cluster._

**Review by cdvultur — COMMENTED** [review]
Overall looks great. Still wondering about the minor.minor version.

File: modules/psap-driver-toolkit-pulling.adoc (hunk 2 above)
Comment (sheriff-rh) [reply modules/psap-driver-toolkit-pulling.adoc:31]: I opted for a variable we use, `{product-version}`, which fills in the viewed version (for example, 4.12, 4.13, and so on). I also just made it `.z` so the user knows to fill in the rest. Thanks!

File: hardware_enablement/about-hardware-enablement.adoc [hunk 3]
```diff
@@ -6,7 +6,6 @@ include::_attributes/common-attributes.adoc[]
...
 
-* The Driver Toolkit is a container image that is a part of every {product-title} release. It contains the kernel packages and other common dependencies that are needed to build a driver or kernel module. The Driver Toolkit can be used as a base image for driver container image builds on {product-title}.
-* The Node Feature Discovery (NFD) Operator adds node labels for CPU capabilities, kernel version, PCIe device vendor IDs, and more.
+Driver containers are container images used for building and deploying out-of-tree kernel modules and drivers on container operating systems like Red Hat Enterprise Linux CoreOS (RHCOS). Kernel modules and drivers are software libraries running with a high level of privilege in the operating system kernel. They extend the kernel functionalities or provide the hardware-specific code required to control new devices. Examples include hardware devices like field-programmable gate arrays (FPGA) or graphics processing units (GPU), and software-defined storage solutions, which all require kernel modules on client machines. Driver containers are the first layer of the software stack used to enable these technologies on {product-title} deployments.
```
Comment (mburke5678) [inline hardware_enablement/about-hardware-enablement.adoc:11]: Per ISG
s/like/such as 

There is a common attribute :op-system-first: Red Hat Enterprise Linux CoreOS (RHCOS)

File: modules/psap-driver-toolkit-pulling.adoc [hunk 4]
```diff
@@ -24,11 +24,15 @@ The driver-toolkit image for the latest minor release will be tagged with the mi
 
 .Procedure
 
-. The image URL of the `driver-toolkit` corresponding to a certain release can be extracted from the release image using the `oc adm` command:
+. The image URL of the `driver-toolkit` corresponding to a certain release are extracted from the release image using the `oc adm` command:
```
Comment (mburke5678) [inline modules/psap-driver-toolkit-pulling.adoc:27]: > Extract the image URL of the `driver-toolkit` corresponding to a certain release from the release image using the `oc adm` command:
??

**Comment by mburke5678** [comment]
A few nits otherwide LGTM

File: hardware_enablement/about-hardware-enablement.adoc (hunk 3 above)
Comment (sheriff-rh) [reply hardware_enablement/about-hardware-enablement.adoc:11]: I often forget about `:op-system-first:`, thank you!

File: modules/psap-driver-toolkit-pulling.adoc (hunk 4 above)
Comment (sheriff-rh) [reply modules/psap-driver-toolkit-pulling.adoc:27]: It sounded better in my head :man_shrugging: :joy:  I'll change it back!

**Pull request merged**